samples/matter/common/src/bridge/bridge_storage_manager.cpp

```diff
@@ -28,7 +33,101 @@ Nrf::PersistentStorageNode CreateIndexNode(uint8_t bridgedDeviceIndex, Nrf::Pers
 
 } /* namespace */
 
-namespace Nrf {
+namespace Nrf
+{
+
+bool BridgeStorageManager::Init()
+{
+	bool result = Nrf::GetPersistentStorage().NonSecureInit();
+
+	if (!result) {
+		return result;
+	}
+
+	/* Perform data migration from previous data structure versions if needed. */
+	result = MigrateData();
+
+	return result;
+}
+
+bool BridgeStorageManager::MigrateData()
+{
+	uint8_t version;
+
+	/* Check if version key is present in settings to provide backward compatibility between post-2.6.0 releases and
```

Comment:
Is the 2.6.1 a one of the post-2.6.0 releases? If so, then the migration is not necessary.

Reply:
No, it's hard to predict how many bug fix releases there will be, so I meant major releases.